src/hotspot/share/jfr/jfr.cpp

```diff
@@ -109,3 +111,10 @@ bool Jfr::on_flight_recorder_option(const JavaVMOption** option, char* delimiter
 bool Jfr::on_start_flight_recording_option(const JavaVMOption** option, char* delimiter) {
   return JfrOptionSet::parse_start_flight_recording_option(option, delimiter);
 }
+
+#ifdef JFR_HAVE_INTRINSICS
```

Comment:
Can make this unconditional from #ifdef JFR_HAVE_INTRINSICS, to simplify a bit.

Reply:
fixed